GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -106,6 +108,17 @@ - (void)updateAuthState:(OIDAuthState *)authState
   _authState = authState;
   _authentication = [[GIDAuthentication alloc] initWithAuthState:authState];
   _profile = profileData;
+  _accessToken = [[GIDToken alloc] initWithTokenString:[self lastTokenResponseAccessToken]
```

Comment:
When initializing instances of `GIDToken` or `GIDConfiguration` we should should `copy` the values.  We're currently doing this in some places and not in others.

Reply:
Done.